app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/core/Declaration.scala

```diff
@@ -217,24 +217,58 @@ case class Goods(categoryQuantityOfGoods: CategoryQuantityOfGoods,
 
     SummaryList(Seq(
```

Comment:
I think its time to refactor this into a component or at least move it into the template.